extension/js/common/composer.ts

```diff
@@ -1090,8 +1090,11 @@ export class Composer {
     }
     if (this.urlParams.isReplyBox) {
       if (this.urlParams.to.length) {
-        this.S.cached('input_text').focus();
-        document.getElementById('input_text')!.focus(); // #input_text is in the template
+        // Chrome needs async focus (#2056)
+        setTimeout(() => {
+          this.S.cached('input_text').focus();
+          document.getElementById('input_text')!.focus(); // #input_text is in the template
+        }, 100);
```

Comment:
It's complaining that if there was an exception thrown inside the anonymous arrow function, the exception would not get reported: a silently dropped error.

We have methods such as `Catch.setHandledTimeout` or something similar. I'll update and merge it.

The err message is not the best.